src/antq/util/maven.clj

```diff
@@ -79,11 +79,23 @@
      :artifact artifact
      :remote-repos remote-repos}))
 
-(defn ^Model read-pom
+(defn- ^Model read-pom*
   [^String url]
   (with-open [reader (io/reader url)]
     (.read (MavenXpp3Reader.) reader)))
 
+(defn ^Model read-pom
+  [^String url]
+  (loop [i 0]
+    (when (< i 5)
+      (or (try
+            (read-pom* url)
+            (catch java.net.ConnectException e
+              (if (= "Operation timed out" (ex-message e))
```

Comment:
Could you change to use `.getMessage` instead of `ex-message` (available since Clojure 1.10) for supporting Clojure 1.8 ?
```suggestion
              (if (= "Operation timed out" (.getMessage e))
```

Reply:
Changed, I did not think of older Clojure versions!